src/features/user/Account/History.tsx

```diff
@@ -172,6 +256,22 @@ export default function History({
               record={paymentHistory.items[selectedRecord]}
             />
           )}
+
+        <Snackbar
+          open={open}
+          autoHideDuration={5000}
+          onClose={handleClose}
+          anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+        >
+          <MuiAlert
+            onClose={handleClose}
+            severity="success"
+            variant="filled"
+            sx={{ width: '100%' }}
+          >
+            {t('me:taxReceiptsSuccess')}
+          </MuiAlert>
+        </Snackbar>
```

Comment:
Perhaps the `CustomSnackbar` component that has already been created can be used here instead?

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/1ae59844cc39bd34ed10b9a92b96d05a54271870